client/me/account-close/confirm-dialog.jsx

```diff
@@ -48,57 +52,156 @@ class AccountCloseConfirmDialog extends React.Component {
 		}
 	};
 
+	handleProceedingToConfirmation = () => {
+		this.setState( { displayAlternativeOptions: false } );
+	};
+
 	handleConfirm = () => {
 		this.props.closeAccount();
 		page( '/me/account/closed' );
 	};
 
+	handleAlternaticeActionClick = ( evt ) => {
+		recordTracksEvent( 'calypso_close_account_alternative_clicked', {
+			type: 'Action Link',
+			label: evt.target.dataset.tracksLabel,
+		} );
+	};
+
 	render() {
 		const { isVisible, currentUsername, translate } = this.props;
-		const isButtonDisabled = currentUsername && this.state.inputValue !== currentUsername;
+		const isDeleteButtonDisabled = currentUsername && this.state.inputValue !== currentUsername;
+
+		const alternativeOptions = [
+			{
+				englishText: 'Start a new site',
+				text: translate( 'Start a new site' ),
+				href: '/jetpack/new',
+				supportLink:
+					'https://wordpress.com/support/create-a-blog/#adding-a-new-site-or-blog-to-an-existing-account',
+				supportPostId: 3991,
+			},
+			{
+				englishText: "Change your site's address",
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'Change my site's address' )` *ES Score: 6*